hw/ip/spi_device/data/spi_device_testplan.hjson

```diff
@@ -180,6 +180,14 @@
       stage: V2
       tests: ["spi_device_mem_parity"]
     }
+    {
+      name: mem_cfg
+      desc: '''Test cfg_i connectivity beween spi_device and prim_ram_2p
+
+               Randomly set dut.ram_cfg_i and check the value same is propagate to prim_mem_2p.'''
```

Comment:
```suggestion
               Randomly set dut.ram_cfg_i and check this value is propagate to prim_mem_2p.'''
```

Reply:
LOL, the earlier sentence doesn't make sense at all... Thanks Cindy. Fixed.